src/main/java/pl/sda/eventmanager/services/UserService.java

```diff
@@ -25,30 +28,33 @@ public UserService(UserRepository userRepository, PasswordEncoder passwordEncode
     @Override
     @Transactional(readOnly = true)
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
+        if(userRepository.findByEmail(email).isPresent())
```

Comment:
use brackets (`{}`) at **all** times, that improves code readability and decreases chance of error while modifying `if`